cpp/oneapi/dal/backend/interop/table_conversion.hpp

```diff
@@ -54,4 +55,18 @@ inline auto convert_to_daal_homogen_table(array<T>& data,
         daal_data, column_count, row_count);
 }
 
+template <typename T>
+inline auto convert_from_daal_homogen_table(const daal::data_management::NumericTablePtr& nt) {
```

Comment:
Please do not use `auto` everywhere. If the type is simple, use the correct type.

Reply:
Ok, thanks!